micrometer-core/src/main/java/io/micrometer/core/instrument/LongTaskTimer.java

```diff
@@ -67,8 +67,7 @@ static Builder builder(Timed timed) {
      * @return The return value of {@code f}.
      * @throws Exception Any exception bubbling up from the callable.
      */
-    @Nullable
-    default <T> T recordCallable(Callable<T> f) throws Exception {
+    default <T> @Nullable T recordCallable(Callable<T> f) throws Exception {
```

Comment:
I'm not sure how to properly mark this. The return value nullability should be the same as the nullability of the `Callable` passed here, which may or may not be nullable.

Reply:
We used `NullUnmarked` for `Observation`, I guess `Timer` has the same issue.

Reply:
I've update it to mark the return type as `@NullUnmarked`